--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,8 +127,16 @@ endif(DEFINED OCE_INCLUDE_PATH)
 # Installation directory                     #
 # by default, installed to site-packages/OCC #
 ##############################################
-execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "from __future__ import print_function; from distutils.sysconfig import get_python_lib; import os;print(get_python_lib())" OUTPUT_VARIABLE python_lib OUTPUT_STRIP_TRAILING_WHITESPACE )
-set(PYTHONOCC_INSTALL_DIRECTORY ${python_lib}/OCC CACHE PATH "pythonOCC install directory" )
+set(GET_SITE_PACKAGE
+  "from distutils.sysconfig import get_python_lib; print(get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}'))")
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+  "${GET_SITE_PACKAGE}" OUTPUT_VARIABLE PY_INSTALL_DIR)
+string(STRIP ${PY_INSTALL_DIR} PY_INSTALL_DIR)
+set(PYTHONOCC_INSTALL_DIRECTORY ${PY_INSTALL_DIR}/OCC CACHE PATH "pythonOCC install directory")
+ 
+ # List of OCE shared libraries to link with
+ # TODO: this should be build from the module list
+
 
 # List of OCE shared libraries to link with
 # TODO: this should be build from the module list
