diff --git a/mechanisms/src/CADMBTB/mymath_FunctionSetRoot.cxx b/mechanisms/src/CADMBTB/mymath_FunctionSetRoot.cxx
index 6fa2e5ef8..c1226ec71 100644
--- a/mechanisms/src/CADMBTB/mymath_FunctionSetRoot.cxx
+++ b/mechanisms/src/CADMBTB/mymath_FunctionSetRoot.cxx
@@ -8,7 +8,7 @@
 //mymath_FunctionSetRoot.cxx
 
 
-#include "mymath_FunctionSetRoot.ixx"
+#include "mymath_FunctionSetRoot.jxx"
 #include <Standard_DimensionError.hxx>
 #include <math_Gauss.hxx>
 #include <math_SVD.hxx>
