--- a/numerics/src/tools/internal/numerics_verbose.h
+++ b/numerics/src/tools/internal/numerics_verbose.h
@@ -85,7 +85,7 @@ extern "C"
   do                                                                    \
   {                                                                     \
     int error_code = EXPR;                                                  \
-    MPI_Errhandler_set(MPI_COMM_WORLD, MPI_ERRORS_RETURN);              \
+    MPI_Comm_set_errhandler(MPI_COMM_WORLD, MPI_ERRORS_RETURN);              \
     if (error_code != MPI_SUCCESS) {                                    \
       char error_string[1024];                                          \
       int length_of_error_string, error_class;                          \
