diff -u VTK-8.2.0.ori/Rendering/FreeType/vtkFreeTypeTools.cxx /home/maurice/src/VTK-8.2.0/Rendering/FreeType/vtkFreeTypeTools.cxx
--- VTK-8.2.0.ori/Rendering/FreeType/vtkFreeTypeTools.cxx	2021-04-09 18:06:17.528165936 +0200
+++ VTK-8.2.0/Rendering/FreeType/vtkFreeTypeTools.cxx	2021-04-09 18:06:48.108976508 +0200
@@ -387,8 +387,7 @@
 }
 
 //----------------------------------------------------------------------------
-FT_CALLBACK_DEF(FT_Error)
-vtkFreeTypeToolsFaceRequester(FTC_FaceID face_id,
+static FT_Error vtkFreeTypeToolsFaceRequester(FTC_FaceID face_id,
                               FT_Library lib,
                               FT_Pointer request_data,
                               FT_Face* face)

Diff finished.  Fri Apr  9 18:08:49 2021
